docs/intermediate/least_to_most.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 5
+locale: en-us
+style: chicago
+---
+
+# 🟢 Least to Most Prompting
+
+Least to Most Prompting (@zhou2022leasttomost) builds upon the idea of Chain-of-Thought prompting and takes it a step further. It is a technique inspired by real-world educational strategies for children.  
+
+As in CoT-Prompting, the problem to be solved is decomposed in a set of subproblems that build upon each other. In a second step, these subproblems are solved individually and in sequence. However, contrary to chain of thought, the solution of previous subproblems is fed into the next question.
+
+import leastToMost from '../assets/least-to-most.png'
```

Comment:
This image doesn't quite represent what LtM does. It is more 1. reduce 2. iteratively solve

Reply:
That's true, although they only use the explicit reduction step in only the SCAN case, in the other two, it's kind of implicit in the way they phrase what looks like a chain of thought at first site. I'll add the optional reduction step.

Reply:
Hmm. Ill take a longer look through the paper.